package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
+    "heroku-prebuild": "cd client && npm i",
```

Comment:
I think it might be better to use `npm ci` here, not sure if that is necessary though

Reply:
I tried it with `npm ci` but it didn't work 🧐